neps/nep-0330.md

```diff
@@ -73,15 +101,22 @@ pub struct Contract {
     pub contract_metadata: ContractSourceMetadata
 }
 
+// NEP structure
+pub struct NEP {
+    pub NEP: u16,
+    pub version: String
+}
```

Comment:
I would avoid restricting people to only specify NEP standards, so just like with the [JSON Events NEP-297](https://nomicon.io/Standards/EventsFormat#events), I would use `standard` (string value that is like `nep171` or it could be `mintbase001`) and `version` (semver is good)

Reply:
```suggestion
// NEP structure
pub struct Standard {
    pub standard: String,
    pub version: String
}
```